platform/amba-alz/architecture_definitions/amba.alz_architecture_definition.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/Azure-Landing-Zones-Library/main/schemas/architecture_definition.json",
+  "name": "alz",
+  "management_groups": [
+    {
+      "archetypes": [
+        "root"
+      ],
+      "display_name": "ALZ root",
+      "exists": false,
+      "id": "alzroot",
+      "parent_id": null
+    },
+    {
+      "archetypes": [
+        "platform"
```

Comment:
similar here, you need to apply both the alz archetype and the amba one

Reply:
We discussed leaving amba standalone, without dependency on alz, as such, this should now only include the amba archetype, right?

For example:

```
    {
      "archetypes": [
        "amba_platform"
      ],
      "display_name": "Platform",
      "exists": false,
      "id": "amba-platform",
      "parent_id": "ambaroot"
    },
```